### DISCUSSION TITLE
Internet Radio Database (streaming url Finder)

### DISCUSSION THREAD

**Discussion by TheBossME** [coffeegreg/YTuner#20] [Ideas]
Hi Greg,

I'm thinking about radio streaming url Finder for ytuner.

The radio-browser.info database was outdated and lots of interested radio stations are missed. 

I need to grab radio websites (shoutcast, radio.de, etc.) which contains the streaming urls and put that information into a pre database, find duplicates and eliminate the dupes.

The other idea was recreate radiobrowser database and update and find duplicates.

What do you think ?. It will be a big project (i know that)

Regards
Beatrice

**Comment by coffeegreg** [comment]
Hi Beatrice

> The radio-browser.info database was outdated and lots of interested radio stations are missed.

However, I noticed that the number of stations on the radio-browser is gradually increasing. :wink: There are probably fewer of those that are in your scope of interest.
I was thinking about the "YTuner Manager" add-on which would enable easy management of your own bookmarks, station lists and categories, and not as currently based on outdated my_stations.ini or yaml files compatible with YCast. It would be possible to display any web sites that provide content about station lists and based on them, the YTuner user would build his own. But I don't know if there's any point in doing this. I have lots of ideas for other projects waiting to be implemented. :smiley: 
When it comes to YTuner, what I miss most is the functionality of on-the-fly transcoding of the AAC/AAC+ audio stream to MP3 format because my RX-V671 does not support AAC and there are many stations that broadcast in this format. :disappointed: I suspect doing this would be more difficult than the `Radio Streaming URL Finder` you proposed and would require a little bit more CPU processing power than my worn-out 400MHz Neoware CA15 host running YTuner. :wink:

**Reply by TheBossME** [reply]
Have you tried something like this ?

mkfifo /tmp/turk.wav
mplayer -vo null -vc null -ao pcm:fast:nowaveheader:file=/tmp/turk.wav http://94.75.222.9:8875

I'm also thinking about streaming proxy 


server {
        listen   443;
        server_name radio.com;
        ....
        location /stream { 
           proxy_pass http://stream.anotherradio.com:8000/mount; 
        }
        ....
}

And pass this through ffmpeg or mplayer etc.
